patches/v8/0007-include-fix-for-compiling-v8-from-workerd.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Orion Hodson <orion@cloudflare.com>
+Date: Mon, 6 Mar 2023 14:53:22 +0000
+Subject: [PATCH] #include fix for compiling v8 from workerd
+
+Changes from #include <> to #include "".
```

Comment:
FWIW we usually fix these issues in bazel by specifying `include_prefix` or `strip_include_prefix` on the build rule, I wonder if that's intended here.

That said it's probably not worth spending much time thinking about unless we try to upstream a fix.

Reply:
Thanks. The file disappears upstream almost immediately after this version (this change is syncing to ew). I found after patching this that someone proposed a similar fix to v8's bazel build.